src/serac/physics/boundary_conditions/tests/serac_boundary_cond.cpp

```diff
@@ -96,11 +96,12 @@ TEST(boundary_cond_helper, element_attribute_dof_list_scalar)
   auto mesh = mfem::Mesh::MakeCartesian3D(4, 4, 4, mfem::Element::HEXAHEDRON);
   mesh.SetAttribute(1, attribute);
   mesh.SetAttribute(31, attribute);
+  mesh.SetAttributes();
```

Comment:
What does this do? The [documentation](https://docs.mfem.org/html/classmfem_1_1Mesh.html#afb720c1a8d716f98e6ed23e6822b2b13) doesn't reveal anything.

Reply:
I asked @jamiebramwell in chat.. she said this:

"you have to force it to update the list"

Reply:
Yep. I'll add a comment.